Simple-Projects/Simple_Projects/Basic/List/List.cpp

```diff
@@ -112,11 +102,10 @@ bool List::operator!=(const List& other) const {
 }
 
 std::ostream& operator<<(std::ostream& out, const List& list) {
-    int last = list.size() - 1;
+    const int last = list.size() - 1;
```

Comment:
Consider using 'const' for variables that need not change. Firstly, the reader will know right away that it's a constant, and secondly, you won't accidentally alter it.